beacon-chain/slasher/chunks.go

```diff
@@ -264,6 +348,45 @@ func (m *MinSpanChunksSlice) Update(
 	return keepGoing, nil
 }
 
+// Update a max span chunk for a validator index starting at a given start epoch, e_c, then updating
+// up to the current epoch according to the definition of max spans. If we need to continue updating
+// a next chunk, this function returns a boolean letting the caller know it should keep going.
+func (m *MaxSpanChunksSlice) Update(
+	chunkIdx uint64,
+	validatorIdx types.ValidatorIndex,
+	startEpoch,
+	currentEpoch,
+	newTargetEpoch types.Epoch,
+) (bool, error) {
```

Comment:
We should really use named return values here and name the boolean keepGoing. Same for MinSpanChunksSlice.Update (actually the documentation of MinSpanChunksSlice.Update says "This function returns a boolean named keepGoing").